python/konduit/client.py

```diff
@@ -10,7 +10,31 @@
 
 class Client(object):
     def __init__(self, timeout=60, input_type='NUMPY', endpoint_output_type='NUMPY',
-                 return_output_type=None, input_names=['default'], output_names=['default'], url=None):
+                 return_output_type=None, url=None):
+
+        try:
+            r = requests.get("{}/healthcheck".format(url))
```

Comment:
Also not really related to this PR, but it's pretty confusing that one needs to specify just a `port` in the serving config and then remember to put it into the `url` when creating the client. maybe we can have a basic to assert that `url.split(os.pathsep)[1]` can be cast to an int etc.

Reply:
Sure. That can be added

Reply:
So as discussed i've separated host and port with host being defaulted to `http://localhost`